lib/archives_sidebar/lib/archives_sidebar.rb

```diff
@@ -30,11 +30,11 @@ def parse_request(_contents, _params)
     article_counts = Content.find_by_sql(["select count(*) as count, #{date_func} from contents where type='Article' and published = ? and published_at < ? group by year,month order by year desc,month desc limit ? ", true, Time.now, count.to_i])
 
     @archives = article_counts.map do |entry|
-      month = (entry.month.to_i%12)+1
+      month = (entry.month.to_i) 
```

Comment:
Added, thanks for the heads up. Didn't even notice the trailing white space.

I think I understand the original logic to it now as well. The original code add +1 to the end of %12 to not get 0. It seems like line 37 month: month-1, was a bit of a red herring. But the point is that you don't even need %12 in the first place and can get rid of these +1 and -1 entirely.